aiida/manage/tests/pytest_fixtures.py

```diff
@@ -521,6 +525,155 @@ def get_code(entry_point, executable, computer=aiida_localhost, label=None, prep
     return get_code
 
 
+@pytest.fixture(scope='session')
+def generate_ssh_key() -> t.Generator[pathlib.Path, None, None]:
```

Comment:
The fixture could also be called `ssh_key` (most of the fixtures "generate" new instances)

Reply:
Well, to be fair, it does actually generate one. But do you mean that when it is called `generate_` the user can call it multiple times to generate multiple instances? I guess that is right. Will change it.